src/pages/[platform]/build-a-backend/graphqlapi/best-practice/query-with-sorting/index.mdx

```diff
@@ -3,7 +3,7 @@ import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
 export const meta = {
   title: 'GraphQL query with sorting by date',
   description: 'How to implement sorting in a GraphQL query',
-  platforms: ['javascript', 'angular', 'nextjs', 'react', 'vue']
+  platforms: ['javascript', 'angular', 'nextjs', 'react', 'vue', 'swift', 'android']
```

Comment:
swift and android weren't included in the platform list but should have been there re: IA Migration spreadsheet